### PR TITLE
Updated projects show view

### PR DESCRIPTION
**Pull request by igorrfc**
Now a warning message is displayed for recurring projects which doesn't contain associated plans:

screen shot 2017-01-20 at 10 40 27 am

The sass was downgraded to **version '3.4.5'** because an error was being raised. The explanation can be found [here](https://github.com/sass/sass/issues/1476)

### CODE REVIEW

File: app/models/plan.rb [hunk 1]
```diff
@@ -9,4 +9,8 @@ class Plan < ActiveRecord::Base
   validates_presence_of :plan_code, :name, :amount, :payment_methods
 
   has_and_belongs_to_many :projects, join_table: 'projects_plans'
+
+  def formatted_amount
+    amount / 100
```
Comment (IgorMarques) [inline app/models/plan.rb]: if it's in cents, shouldn't it be `* 100`?

Comment (IgorMarques) [inline app/models/plan.rb]: Also, maybe using `in_cents` for the method name would be better

Comment (igorrfc) [inline app/models/plan.rb]: The value is returned in cents, @IgorMarques . So, to format for reais I need to divide for 100, right?

Comment (IgorMarques) [inline app/models/plan.rb]: Oh, I see. I thought you wanted to display the amount in cents. My bad

Comment (igorrfc) [inline app/models/plan.rb]: No problem 😄

**Pull request merged**